modules/sso/sso-pre-launch-checks/added/sso_image_pre_launch_checks.sh

```diff
@@ -60,3 +61,19 @@ function verify_KEYCLOAK_16736_fix_present() {
     exit "${errorExitCode}"
   fi
 }
+
+# CIAM-1757:
+#
+# Confirm JDK 1.8 rpms aren't present in the image, since using JDK 11 already
+#
+function verify_CIAM_1757_fix_present() {
+  # shellcheck disable=SC2207
+  declare -a JDK_18_RPMS=($(rpm --query --all name=java* version=1.8.0*))
+  local -r errorExitCode="1"
+  if [ "${#JDK_18_RPMS[@]}" -ne "0" ]
```

Comment:
Similar to my comment on configure.sh, the array assignment isn't really needed. A pipe can also be used here to the same effect:

```bash
if rpm --query --all name=java* version=1.8.0* | grep -q '.'
then
```

`grep -q '.'` returns true if there's anything on the input, or false if there is nothing

Could also use:

```
if [ -n "$(rpm --query --all name=java* version=1.8.0*)" ]
then
```

Reply:
Used the 2-nd form, since it seemed a bit more readable to me. Thanks for the improvement